torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx

```diff
@@ -128,6 +128,13 @@ function HudRow({
         {rowData.prNum !== null && (
           <a
             href={`https://github.com/${params.repoOwner}/${params.repoName}/pull/${rowData.prNum}`}
+            title={
+              rowData.isForcedMerge
+                ? rowData.isForcedMergeWithFailures
+                  ? "Forced merge with failures"
+                  : "Forced merge"
```

Comment:
Nice idea! And we can make these even more descriptive:

```suggestion
                  ? "Forced merge with failures that were merge-blocking"
                  : "Forced merge. Had no merge-blocking failures"
```